types/trusted-types/index.d.ts

```diff
@@ -60,16 +60,39 @@ declare global {
         // people handle the case when their code is running in a browser not
         // supporting trustedTypes.
         trustedTypes?: TrustedTypePolicyFactory;
-        TrustedHTML: TrustedHTML;
-        TrustedScript: TrustedScript;
-        TrustedScriptURL: TrustedScriptURL;
+        TrustedHTML: typeof TrustedHTML;
+        TrustedScript: typeof TrustedScript;
+        TrustedScriptURL: typeof TrustedScriptURL;
         TrustedTypePolicyFactory: TrustedTypePolicyFactory;
         TrustedTypePolicy: TrustedTypePolicy;
     }
 }
 
-// This is not available in global scope. It's only used for the export. This is
-// necessary to be able to use these types from nodejs (for SSR).
-declare const trustedTypes: TrustedTypePolicyFactory;
+// These are the available exports when using the polyfill as npm package (e.g. in nodejs)
+interface InternalTrustedTypePolicyFactory extends TrustedTypePolicyFactory {
+  TrustedHTML: typeof TrustedHTML;
+  TrustedScript: typeof TrustedScript;
+  TrustedScriptURL: typeof TrustedScriptURL;
+}
+
+declare const trustedTypes: InternalTrustedTypePolicyFactory;
+
+declare class TrustedTypesEnforcer {
+    constructor(config: TrustedTypeConfig);
+    install: () => void;
+    uninstall: () => void;
+}
+
+// tslint:disable-next-line
```

Comment:
what is this disabling ?

Reply:
https://palantir.github.io/tslint/rules/no-unnecessary-class/

I've modified the disable for this rule only and for more context.